sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/kafka/jaas/JaasResolver.java

```diff
@@ -0,0 +1,20 @@
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.implementation.kafka.jaas;
+
+import org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule;
+
+import static com.azure.spring.cloud.service.implementation.kafka.AzureKafkaPropertiesUtils.convertJaasStringToMap;
+
+public final class JaasResolver {
+
+	private JaasResolver() {
+	}
+
+	public static Jaas resolve(String jaasConfig) {
+		Jaas jaas = new Jaas(OAuthBearerLoginModule.class.getName());
```

Comment:
again, you're really coupling things together.

Kafka uses Jaas for its authentication, right?
so, JaaS itself doesn't know anything about kafka, right?